### PR TITLE
fix: relationship docs

### PR DESCRIPTION
**Pull request by RipulHandoo**
**Notes for Reviewers**

This PR:
1. Get down to single link (i.e [Model](https://docs.meshery.io/project/contributing/contributing-models))
2. Add HTML tags to code block in .md file

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/concepts/logical/relationships.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ redirect_from:
 - concepts/relationships
 ---
 
-[Relationships](https://github.com/meshery/meshery/tree/master/server/meshmodel/relationships) define the nature of interaction between interconnected components in Meshery. They represent various types of connections and dependencies between components no matter the genealogy of the relationship such as parent, siblings, binding. Relationships have selectors, metadata, and optional parameters.
+[Relationships](https://docs.meshery.io/project/contributing/contributing-models) define the nature of interaction between interconnected components in Meshery. They represent various types of connections and dependencies between components no matter the genealogy of the relationship such as parent, siblings, binding. Relationships have selectors, metadata, and optional parameters.
```
Comment (leecalcote) [inline docs/pages/concepts/logical/relationships.md:13]:
```suggestion
[Relationships]({{site.baseurl}}/project/contributing/contributing-models) define the nature of interaction between interconnected components in Meshery. They represent various types of connections and dependencies between components no matter the genealogy of the relationship such as parent, siblings, binding. Relationships have selectors, metadata, and optional parameters.
```

Comment (leecalcote) [reply docs/pages/concepts/logical/relationships.md:13]: @RipulHandoo use the macro for these references, please.

Also, this is the wrong place to put the contributing link.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/concepts/logical/relationships.md (hunk 1 above)
Comment (RipulHandoo) [reply docs/pages/concepts/logical/relationships.md:13]: @leecalcote If that's the case, then there's no need to include any other links here.

**Comment by RipulHandoo** [comment]
This issue being addressed [here](https://github.com/meshery/meshery/pull/10253).

File: docs/pages/concepts/logical/relationships.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/concepts/logical/relationships.md:13]: @RipulHandoo, yes, there is need for this, just not in the way that you're doing it here. How would anyone know that this goes to contributing and not to a further explanation of the concept?

**Pull request closed without merging**